### ISSUE TITLE
Different physics outcomes when using the gripper on different scales

### ISSUE THREAD

**Issue by ProdyParrot** [Unity-Technologies/articulations-robot-demo#41]
The gripper gives the desired behavior when used at its original size, but when it is scaled down, it doesn't seem to be able to grip other objects properly.

https://user-images.githubusercontent.com/536207/143979408-25dc3a42-628f-4764-ad7d-54940489bf24.mp4

**Comment by at669** [comment]
Hi, thanks for checking this out! I've opened an internal ticket to investigate this and we'll get back to you with any updates.

[Ticket#: AIRO-1608]

**Comment by JuozasK** [comment]
Hey! Could you report a bug on this with the bug reporter and attach the repro project? That way we can investigate it better! Thank you!

**Comment by ProdyParrot** [comment]
> Hey! Could you report a bug on this with the bug reporter and attach the repro project? That way we can investigate it better! Thank you!

Thanks for the reply! Here is a version of the project with a modified GripperScene and some minor changes to the input scripts to replicate the video in the original post.

The desired behavior is the physics outcomes on both grippers being the same, despite the scale. I have done further experimentation on my own and it seems that reducing the gravity in the physics settings helps to mitigate the undesired behavior of the object falling from the smaller gripper, but it can still happen if the gripper's speed is set to a high value.

[ArticulationRobot.zip](https://github.com/Unity-Technologies/articulations-robot-demo/files/7653706/ArticulationRobot.zip)

**Comment by JuozasK** [comment]
Hey! I took a look at your project and had to make some changes for the robot to actually react to controls. 

Otherwise, I noticed the smaller cube had the same mass as the large one, so that definitely should be reduced based on the scale you want to have

Also, there were some ghost contacts happening because the contact offset value (default value can be found in project settings) does not match the scale anymore and should be reduced as well. 

For the smaller scale I would expect there should be a requirement of higher precision so you might want to increase the solver iterations count.

From my tests it helped the robot grip the smaller cube, though it wasn't a 1 to 1 match with the big cube. (Though I wouldn't expect it to be exact to be honest)
